tests/roots/test-intl/versionchange.txt

```diff
@@ -14,3 +14,6 @@ i18n with versionchange
 .. versionchanged:: 1.0
 
    This is the *first* paragraph of versionchanged.
+
+.. versionremoved:: 1.0
+   This is the *first* paragraph of versionremoved.
```

Comment:
Maybe worth testing the oneliner form too, since it isn't tested here?

```suggestion
.. versionremoved:: 1.0 This is the *first* paragraph of versionremoved.
```